snippets/frameworks/angular/dockerfile.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Multi-stage Node and Angular Dockerfile": {
```

Comment:
Similar to my other comment, this seems like it could be omitted from this PR (and retain the focus on adding the Angular snippets).